### ISSUE TITLE
bug exception

### ISSUE THREAD

**Issue by michaldengusiak** [HoareLea/SAM_Deploy#6]
issue on some PCs

![image](https://user-images.githubusercontent.com/26113670/90764120-c2159480-e2df-11ea-8e4c-b62a4fb3a457.png)

**Comment by willhl** [comment]
Hmm...
![image](https://user-images.githubusercontent.com/17121741/90832940-d5a41800-e33e-11ea-9eba-c032c0828e6b.png)
Left: C:\Program Files\Rhino 6\Plug-ins\Grasshopper
Right: Nuget
Build differs very slightly.

**Comment by willhl** [comment]
Nope! I changed the refs over from nuget to local ones... same issue...

**Comment by willhl** [comment]
OK I've been down a few rabbit holes and I think I've found the issue.. but fixing it is a pain.

It could be because we have SAM.Core.Grasshopper.gha and SAM.Core.Grasshopper.dll present, I think when it loads a .gha which depends on another .gha, the assembly resolver finds the .dll and loads it, so now it's got the .gha version of the same library loaded too. I thought I could fix it by explicitly setting the load order in the .ghlink file, it didn't work, still!
